qa/common/rbd.sh

```diff
@@ -0,0 +1,22 @@
+#
+# This file is part of the DeepSea integration test suite
+#
+
+function ceph_conf_rbd_default_features_61 {
+  sed -i 's/rbd default features =.*/rbd default features = 61/g' \
```

Comment:
I would just remove this line from the file. Then the current Ceph version "rbd default features" would be used.